### PR TITLE
ESET Package

### PR DESCRIPTION
**Pull request by tomdamonlz**

### CODE REVIEW

File: packages/ESET-Management-Console/syslog-ng/eset-syslog.conf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+filter f_program {program("lzadmin")};
```
Comment (claytondukes) [inline packages/ESET-Management-Console/syslog-ng/eset-syslog.conf]: lzadmin?

Comment (tomdamonlz) [reply packages/ESET-Management-Console/syslog-ng/eset-syslog.conf]: Could have sworn I changed that.

**Comment by tomdamonlz** [comment]
I don't have any config info for the server.

File: packages/ESET-Management-Console/syslog-ng/eset-syslog.conf [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+log {
+    source(s_logzilla);
+    rewrite (r_quotes);
+    #filter(f_fwdrops);
+    destination(d_logzilla_network);
+    # Uncomment line below for debug/testing of incoming events
+    #destination(df_debug);
+    #destination(d_unix_stream);
+    flags(flow-control,final);
+};
```
Comment (claytondukes) [inline packages/ESET-Management-Console/syslog-ng/eset-syslog.conf]: This will fail if they already have a custom syslog-ng config

**Comment by MG818** [comment]
In gjerstad email account

_____________________________________________

**Pull request closed without merging**